### PR TITLE
Add SafeBuffer support (up to 6.1.0)

### PR DESCRIPTION
**Pull request by DmitryAstafyev**
## Reason
No way to use `ArrayBuffer` in context of `electron`

## Subject

Adds supporting of `SafeBuffer`. Unfortunately `electron` has restrictions to use `ArrayBuffer` as a workaround can be used `SafeBuffer`.

## Realations

- related to #309

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "node-bindgen"
-version = "6.0.2"
+version = "6.0.3"
```
Comment (sehz) [inline Cargo.toml:3]: Can you change to "6.1.0" since its more than patch?

**Review by sehz — CHANGES_REQUESTED** [review]
LGTM.  A small nit on version number

File: Cargo.toml (hunk 1 above)
Comment (DmitryAstafyev) [reply Cargo.toml]: @sehz done

**Comment by sehz** [comment]
Oh. You hit the jackpot, It's rust update date today.    Can you create another PR to update clipping rules?  Thanks

**Comment by DmitryAstafyev** [comment]
Some clippy errors, will fix and include

**Comment by DmitryAstafyev** [comment]
@sehz done

**Review by sehz — APPROVED** [review]
Thanks.  Great job

**Pull request merged**